recipes/jupyter-collaboration/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "jupyter-collaboration" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/jupyter_collaboration-{{ version }}.tar.gz
+  sha256: 7a79651d17898564fe13ad4c31bde50059e7087346339045159126e64dd565bd
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8
+    - pip
+    - hatchling >=1.4.0
+    - hatch-nodejs-version
+    - hatch-jupyter-builder
+    - jupyterlab >=4.0.0
+  run:
+    - python >=3.8
+    - jupyter_server >=2.0.0,<3.0.0
+    - jupyter_ydoc >=1.0.1,<2.0.0
+    - ypy-websocket >=0.8.3,<0.9.0
+    - jupyter_events
+    - jupyter_server_fileid >=0.6.0,<1
+
+test:
+  imports:
+    - jupyter_collaboration
+  requires:
+    - pip
+  commands:
+    - pip check
```

Comment:
recommend verifying _at least_ e.g. `labextension list`, `serverextension list`: this has caught real problems on other projects that switched up their `build_system`, etc. But really would recommend running at least _some_ of the test suite.